validation-test/compiler_crashers_fixed/28505-failed-call-arguments-did-not-match-up.swift

```diff
@@ -5,6 +5,6 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: not --crash %target-swift-frontend %s -emit-ir
+// RUN: not %target-swift-frontend %s -emit-ir
```

Comment:
Sure, yeah, but i think it general it's better not to allow the unresolved types in there since there is nothing going to diagnose things further and we are missing out a bunch of diagnostic opportunities that way...